hs_geo_raster_resource/templates/pages/rasterresource.html

```diff
@@ -113,10 +102,23 @@
                                     <strong>{{ band.name }}</strong>
                                     <table class="custom-table">
                                         <tbody>
+
                                             <tr><th class="text-muted">VariableName</th><td>{{ band.variableName }}</td></tr>
 
                                             <tr><th class="text-muted">VariableUnit</th><td>{{ band.variableUnit }}</td></tr>
 
+                                            {% if band.noDataValue != None%}
```

Comment:
@hyi Sometimes the no data value, max and min values will be 0. Under these conditions, the value won't be shown in view mode of the resource landing page if I use {% if band.noDataValue %}